Examples/LambdaFunctions/Sources/LambdaAPI/swagger.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
used?

Reply:
It's the Swagger you can open with PostMan to have the API definition and test it. I need to insert this in some README. Do you have suggestion for it?